packages/hello-gsm-admin/src/Utils/Libs/requestUrls.ts

```diff
@@ -1,34 +1,55 @@
 // 회원
 export const AuthController = {
-  signin: () => {
-    return `/auth/login`;
-  },
+  // 각 oauth 로그인 URL 반환
+  signin: {
+    kakao: () => {
+      return `/auth/v1/oauth2/authorization/kakao`;
+    },
+    google: () => {
+      return `/auth/v1/oauth2/authorization/google`;
+    },
+    github: () => {
+      return `/auth/v1/oauth2/authorization/github`;
+    },
+  },
+  // logout URL 반환
   logout: () => {
-    return `/auth/logout`;
+    return `/auth/v1/logout`;
   },
-  refresh: () => {
-    return `/auth/refresh`;
+};
+
+// 유저 상태
+export const UserController = {
+  // get 특정 사용자 정보 조회하기
+  userInfo: (userId: string) => {
+    return `/user/v1/user/${userId}`;
   },
-  check: () => {
-    return `/auth/check`;
+};
+
+// 유저 신원
+export const IdentityController = {
+  // get 특정 사용자의 본인인증 정보 조회하기
+  getUserIdentity: (userID: string) => {
+    return `/identity/v1/identity/${userID}`;
```

Comment:
컨벤션에 맞게 userId로 수정하는게 좋을 것 같습니다!
```suggestion
  getUserIdentity: (userId: string) => {
    return `/identity/v1/identity/${userId}`;
```

Reply:
suggestion 반영 하였습니다! 말씀 감사드립니다!!